app-editors/qhexedit2/qhexedit2-0.8.6.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Use the latest available EAPI. 7 should work.

Reply:
done